code/modules/events/slaughterevent.dm

```diff
@@ -36,7 +36,7 @@
 		kill()
 		return
 	var/obj/effect/dummy/slaughter/holder = new /obj/effect/dummy/slaughter(pick(spawn_locs))
-	var/mob/living/simple_animal/slaughter/S = new /mob/living/simple_animal/slaughter/(holder)
+	var/mob/living/simple_animal/slaughter/lesser/S = new /mob/living/simple_animal/slaughter/lesser/(holder)
```

Comment:
Please make two of this event that's slaughter lesser and slaughter greater.

It'd be nice to have a generic event that just pulls a random person in to be slaughter demon--sometimes its useful for events and ragin' mages shenanigans.

Reply:
Done!